webpack.config.js

```diff
@@ -66,7 +66,7 @@ const baseConfig = {
                 ],
             },
             {
-                test: /node_modules[\\|/](amazon-states-language-service|vscode-json-languageservice)/,
+                test: /node_modules[\\|/](amazon-states-language-service|vscode-json-languageservice|yaml-language-server)/,
                 use: { loader: 'umd-compat-loader' },
             },
```

Comment:
We can delete all of this. This is probably causing the v8 heap out-of-memory fault

Reply:
Thats what I was thinking, I'm just a bit confused why it didn't have the issue with vscode-json-languageservice but had it with the yaml one 🤔

Reply:
Loader is fine, I was looking at the wrong build script. 
We need to change `.github/workflows/node.js.yml` and move `NODE_OPTIONS: '--max-old-space-size=8192'` so it covers all commands for Mac and Windows.